README.md

```diff
@@ -55,38 +57,43 @@ paul.add_post(TextPost("Post 1"))
 george.add_post(TextPost("Post 2"))
 paul.add_post(TextPost("Post 3"))
 
+print(john.following)
+>>>[[<User: "Paul McCartney">], [<User: "George Harrison">]] # John is following Paul and George
+
 print(john.get_timeline())
->>> [<TextPost: Post 3>, <TextPost: Post 2>, <TextPost: Post 1>
+>>> [<TextPost: Post 3>, <TextPost: Post 2>, <TextPost: Post 1> 
 ```
 
+**Important**: A user's timeline should only include posts from the users they are following.
+
 ## Reading Posts
 
 Finally, one of the most interesting use cases of this project is going to be realted to the "visual representation" of the posts. It's a great example of [Polymorphism](https://en.wikipedia.org/wiki/Polymorphism_(computer_science)). The concept is simple. If I try to print different types of posts, I'm going to get different representations. Example:
 
 
 ```python
 john = User("John", "Lennon", "john@rmotr.com")
-post_1 = TextPost("All you need is love!")
-post_2 = PicturePost("Check my new submarine.", image_url='imgur.com/submarine.jpg')
-post_3 = CheckInPost("At Abbey Road Studios", latitude="19.111", longitude="-9.2222")
-john.add_post(post_1)
-john.add_post(post_2)
-john.add_post(post_3)
-
-print(post_1)
+text_post_1 = TextPost("All you need is love!")
+picture_post_2 = PicturePost("Check my new submarine.", image_url='imgur.com/submarine.jpg')
+checkin_post_3 = CheckInPost("At Abbey Road Studios", latitude="19.111", longitude="-9.2222")
```

Comment:
I we change to more descriptive variables, we might be able to do away with the number designations. Unless we go with number 9 for everything. Then I'm completely on board.